images/common/install-sambacc-common.sh

```diff
@@ -31,26 +31,48 @@ install_sambacc() {
 
 
     local action=install-from-copr-repo
-    if [ "${#wheels[@]}" -gt 1 ]; then
-        echo "more than one wheel file found"
-        exit 1
-    elif [ "${#wheels[@]}" -eq 1 ]; then
-        action=install-wheel
+    local source_found=0
+    if [ "${#wheels[@]}" -gt 0 ]; then
+        echo "INFO: wheel found"
+        if [ "${#wheels[@]}" -gt 1 ]; then
+            echo "ERROR: more than one wheel file found"
+            exit 1
+        elif [ "${#wheels[@]}" -eq 1  ]; then
+            echo  "sambacc wheel found. Installing wheel."
```

Comment:
I think these lines (like `echo  "sambacc wheel found. Installing wheel."` and ` echo "sambacc rpm found. preferring  rpm install."` really ought to be saved until *after* the if statements have all been processed. Otherwise, we will get results where it might print something like:

```
INFO: wheel found
sambacc wheel found. Installing wheel.
INFO: rpm file found
sambacc rpm found. preferring  rpm install.
INFO: repo file found"
sambacc repo file found. preferring  install from local yum repo.
```

If I didn't know the code well, I'd find that confusing. I think the `INFO:`  lines are good, they're true. But I'd rather not say anything about (preferred) install method until after the final assignment of `action`.

Reply:
agreed. Will address tomorrow.

Reply:
done